CMakeLists.txt

```diff
@@ -33,7 +33,14 @@ endif()
 option (REPO_BUILD_COVERAGE "Building with coverage report" OFF)
 option (ODA_SUPPORT "If ODA library is present" OFF)
 option (SYNCHRO_SUPPORT "If Synchro library is present" OFF)
-option (VALID_UNTIL "Set timestamp for how long (sec since epoch) this library should be valid for (-1 for unlimited) " -1)
+
+option (LICENSE_CHECK  "Build with a license check, see for details: https://github.com/3drepo/wiki/wiki/Licensing-components-of-3drepo.io " OFF)
+option (LICENSE_RSA_PUB_KEY_MOD "Licensing public token modulus" "")
```

Comment:
these should only show if `LICENSE_CHECK` is on. move it to L126

`option` also probably isn't the right syntax, as the documentation depicts it to be an on and off thing. https://cmake.org/cmake/help/latest/command/option.html

we should be using set https://cmake.org/cmake/help/latest/command/set.html